packages/ui/src/radio/RadioField.tsx

```diff
@@ -1,12 +1,32 @@
 import {
-  FormControl,
+  FormControl as FormControlMui,
   FormControlLabel,
   FormControlLabelProps,
-  FormHelperText,
+  FormHelperText as FormHelperTextMui,
   Radio,
   RadioProps,
 } from '@material-ui/core';
 import { forwardRef, ForwardRefExoticComponent, ReactNode } from 'react';
+import styled from 'styled-components';
+import { Color } from '../theme/Color';
+
+const FormControl = styled(FormControlMui)`
+  justify-content: center;
+`;
+
+const FormHelperText = styled(FormHelperTextMui)`
+  color: ${Color.Dark300};
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 400;
+  margin-left: 30px;
+  margin-top: 0;
+  margin-bottom: 4px;
```

Comment:
@umrzoq-toshkentov Have you shown changes to the design team? You can also install new packages locally from the codesandbox, for example. And check how existing forms will be changed.

Reply:
@timurhamitov yes of course after merge I will try it

Reply:
@umrzoq-toshkentov You can also try it without merging by using [links](https://ci.codesandbox.io/status/superdispatch/web-ui/pr/483/builds/414630) from [these comment](https://github.com/superdispatch/web-ui/pull/483#issuecomment-1699050709). There will be `yarn add` command example: 
<img width="1440" alt="Screenshot 2023-09-05 at 11 44 07" src="https://github.com/superdispatch/web-ui/assets/4053068/f5ddddf2-f734-419a-ad3d-a55152eb3158">

Reply:
@timurhamitov thank you I have tested it on my local. Everything works as expected